localization/en.po

```diff
@@ -207,3 +207,115 @@ msgstr " The game will finish at the end of the round."
 
 msgid "Только хост может закончить игру"
 msgstr "The host only can finish the game"
+
+msgid "ты"
+msgstr "you"
+
+msgid "Шляпа"
+msgstr "TheTrueHat"
+
+msgid "Играть"
+msgstr "Play"
+
+msgid "Слова"
+msgstr "Words"
+
+msgid "Загрузить"
+msgstr "Load"
+
+msgid "Количество слов"
+msgstr "Total number of words"
+
+msgid "ход"
+msgstr[0] "turn"
+msgstr[1] "turns"
+
+msgid "пока не кончатся слова"
+msgstr "until the words are over"
+
+msgid "заданное число кругов"
+msgstr "fixed number of rounds"
+
+msgid "От каждого игрока"
+msgstr "From each player"
+
+msgid "Выбрать файл"
+msgstr "Choose file"
+
+msgid "Файл не выбран"
+msgstr "No file chosen"
+
+msgid "Количество кругов"
+msgstr "The number of rounds"
+
+msgid "Загрузите текстовый файл со словами, каждое слово должно быть на отдельной строке."
+msgstr "Upload the text file with words, each word must be written on a separate line."
+
+msgid "Исходное количество слов в шляпе, выбранных из словаря."
+msgstr "The initial number of words from dictionary in hat."
+
+msgid "Количество кругов"
+msgstr "The number of rounds"
+
+msgid "За один круг каждый игрок будет в паре с каждым из остальных игроков ровно по два раза: как объясняющий и как отгадывающий."
+msgstr "In one round each player will be in pair with each of the rest exactly twice: as speaker and as listener."
+
+msgid "Ты объясняешь на следующем ходу"
+msgstr "You explain the next turn"
+
+msgid "Ты угадываешь на следующем ходу"
+msgstr "You guess the next turn"
+
+msgid "Ты будешь объяснять через"
+msgstr "You explain in"
+
+msgid "Ты будешь угадывать через"
+msgstr "You guess in"
+
+msgid "Нет соединения, перезагрузите страницу"
+msgstr "No connection, reload the page"
+
+msgid "Нужно выбрать файл"
+msgstr "You should send a text file"
+
+msgid "загрузка"
+msgstr "loading"
+
+msgid "Файл загружается"
+msgstr "File is uploading"
+
+msgid "Невозможно открыть файл"
+msgstr "Can't open file"
+
+msgid "Необходимо загрузить текстовый файл"
+msgstr "You should send a text file"
+
+msgid "Максимальный размер словара 64 Mб"
+msgstr "Max dictionary size is 64 MiB"
+
+msgid "Нужно ввести ключ комнаты"
+msgstr "You should enter the room’s key"
+
+msgid "Нужно ввести имя"
+msgstr "You should enter the name"
+
+msgid "Такое имя уже занято"
+msgstr "This name is engaged"
+
+msgid "Чтобы войти в игру нужно ввести имя, с которым вы играли."
+msgstr "You should enter name with which you played to enter the room."
+
+msgid "Набор слов"
+msgstr "Entering words"
+
+msgid "Вводите слова в поле ниже. Одно слово - одна строчка. Если одно и тоже слово введут несколько человек, в шляпе оно встретится ровно один раз."
+msgstr "Write words in box below. Each word must be written on a separate line. If several people write the same word, it will be appeared only once."
```

Comment:
1. Вот тут как раз "enter" - корректный перевод "вводите" вместо "write".
1. Смущает, что слово "поле" переведено как "box", а не "field". Но это уже чисто на усмотрение (я не специалист).
1. "It will be appeared". "Be appeared". Шта?! "Appear" - и так "появляться (самому!)". А так его зачтавлют самому появляться. Что, чёрт возьми?!
1. Ещё формальности: "exactly once".
```suggestion
msgstr "Enter words in box below. Each word must be written on a separate line. If several people write the same word, it will appear exactly once."
```

Reply:
Что только не напишешь в 11 ночи... Изменения одобряю.
Про field и box: я формальной разницы не знаю, но по ощущениям field обычно меньше, чем box. В нашем случае поле довольно большое.